src/Evolution/Systems/ScalarWave/BoundaryConditions/SphericalRadiation.cpp

```diff
@@ -0,0 +1,120 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/ScalarWave/BoundaryConditions/SphericalRadiation.hpp"
+
+#include <cstddef>
+#include <memory>
+#include <pup.h>
+
```

Comment:
probably want `<optional>` `<string>`, `"Domain/BoundaryConditions/BoundaryConditions.hpp"` `"Utilities/Gsl.hpp"`

Reply:
I have yet to figure out/decide if having them in the corresponding `hpp` should be sufficient, but I'll add them